src/graphql/mutations.js

```diff
@@ -0,0 +1,21 @@
+import { gql } from 'graphql-tag';
+
+const REGISTER = gql`
+mutation REGISTER(
+  $username: String!
+  $firstName: String!
+  $lastName: String!
+  $password: String!
+) {
+  createUser(
+    username: $username
+    firstName: $firstName
+    lastName: $lastName
+    password: $password
+    userRoleId: 3
+  ) {
+    id
+  }
+}`;
```

Comment:
I think you're missing indentation here, not sure why prettier didn't suggest it for you:
```
const REGISTER = gql`
  mutation REGISTER($username: String!, $firstName: String!, $lastName: String!, $password: String!) {
    createUser(username: $username, firstName: $firstName, lastName: $lastName, password: $password, userRoleId: 3) {
      id
    }
  }
`;
```